pkg/products/monitoring/reconciler.go

```diff
@@ -805,7 +805,10 @@ func (r *Reconciler) reconcileAlertManagerConfigSecret(ctx context.Context, serv
 	smtpAlertFromAddress := os.Getenv(integreatlyv1alpha1.EnvKeyAlertSMTPFrom)
 
 	// If SMTPFromAddress already set for RHMI prod customers, do not reset
-	if r.installation.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManaged) && existingSMTPFromAddress != "" {
+
+	if r.installation.Spec.Type == string(integreatlyv1alpha1.InstallationTypeMultitenantManagedApi) {
```

Comment:
Not a big deal but would it be better to re-use existing logic to check installation type:
```
integreatlyv1alpha1.IsRHOAMMultitenant(integreatlyv1alpha1.InstallationType(installation.Spec.Type)) {
		smtpAlertFromAddress = "noreply-test@rhmw.io"
} else if integreatlyv1alpha1.IsRHOAM(integreatlyv1alpha1.InstallationType(inst.Spec.Type)) existingSMTPFromAddress != "" {
....
```